renderer/backend/metal/command_buffer_mtl.mm

```diff
@@ -53,6 +53,18 @@
   return CommandBufferMTL::Status::kError;
 }
 
+// TODO(dnfield): remove this declaration when we no longer need to build on
+// machines with lower SDK versions than 11.0.s
+#if !defined(MAC_OS_X_VERSION_11_0) || MAC_OS_X_VERSION_MAX_ALLOWED < MAC_OS_X_VERSION_11_0
```

Comment:
What exactly are you trying to do?  Does `API_AVAILABLE(macos(11.0))` work?  Do you need an iOS availability constraint as `MTLCommandEncoderErrorStateToString` has `API_AVAILABLE(ios(14.0), macos(11.0))`

Reply:
See more of this continued in #136.

I'm trying to make sure that if we end up on a bot that has an older SDK that does not define this enum, we can still compile.